### ISSUE TITLE
The file is not even getting opened in android studio.Tell me how to run?

### ISSUE THREAD

**Issue by BrandedAlok** [googlesamples/android-BluetoothChat#15]
please help

**Comment by mainr** [comment]
Assum you have either download the project as a ZIP and unzip to a directory, OR cloned the repository with GIT. Either way, the android-BluetoothChat-master will exist on your local drive.

To open / build,
1. Start Android Studio.
2. If you have an existing project open in Android Studio, close it.
3. From the "Welcome to Android Studio' dialog, select "Open an existing Android Studio project".
4. Navigate to the directory where 'android-BluetoothChat-master' exists, and select "OK".
5. Resolve any dependency errors (For instance, I had to download the latest build tools and re-sync).

**Comment by BrandedAlok** [comment]
![hmm1](https://cloud.githubusercontent.com/assets/21073030/17765336/d62cb86e-6542-11e6-8651-c58b402510ed.jpg)
![hmm2](https://cloud.githubusercontent.com/assets/21073030/17765338/d91e5b40-6542-11e6-81df-6e793baf7a8c.jpg)
i did sir .
When i m opening it is showing nothing in the project structure

**Comment by BrandedAlok** [comment]
I am not even able to click on the "Run Button " as it is disabled :((

**Comment by mainr** [comment]
I've removed & re-imported this project multiple times & ways without any problems, so I have to assume there is something mis-configured / confused / non-standard on your installation of Android Studio.

Please remove all trace of the existing imported project (delete from list on left of "Welcome to Android Studio", and delete from disk. Either extract from ZIP again, or re-clone the repository, so you have a virgin image. They instead of importing the project with "Open an existing Android Studio project" from the "Welcome to Android Studio" dialog, import the project with "Import project (Eclipse ADT, Gradle, etc.)".

Hopefully this will fix whatever the incompatibility is.

Good Luck.

**Comment by BrandedAlok** [comment]
Okay sir thank you project is opening and running.
But how to send chat?
when i m sending message to the device i connected with bluetooth, the message is not received in the second device !!
please tell how to send and see the messages which i have send into the other mobile device through which bluetooth is connected!
Do i haveto install this app on both the mobiles?

**Comment by mainr** [comment]
Yes, the app must be installed on both devices.
If it is working properly, you will see tagged messages from each device on both screens.

**Comment by BrandedAlok** [comment]
how to install on both the device?
can i send using shareit after launching in one mobile because how can i
connect both the mobiles at same time?

**Comment by mainr** [comment]
I assume you are using Android Studio to "RUN" the application and selecting the mobile phone you have connected by USB as the target. Once you have the app installed / running in this fashion on the first device, unplug the USB cable, connect it to the second device, and "RUN" the application on the second device in the same fashion. The application continues to run on the first device even with the USB cable removed.

**Comment by BrandedAlok** [comment]
Yes sir now it is working perfectly sir . Thanking you so much warm regards and thank you so much for replying.
I liked your efforts and your skills of Android Development. :100: :+1:

**Comment by WildeHa** [comment]
Just a quick guess, did you open it from "Open" or "Import"? if it was "Open" then you could try "Import" the project in your PC instead.
